guides/common/modules/proc_using-vmware-cloud-init-and-userdata-templates-for-provisioning.adoc

```diff
@@ -26,18 +26,17 @@ When you set up the compute resource and images for VMware provisioning in {Proj
 * VMware vCenter applies customization for the virtual machine's identity, including the host name, IP, and DNS
 * The virtual machine builds, `cloud-init` is invoked and calls back {Project} on port `80`, which then redirects to `443`
 
-.Port and Firewall Requirements
+.Prerequisites
+* Ensure that you configure port and firewall settings to open any necessary connections.
 Because of the `cloud-init` service, the virtual machine always calls back to {Project} even if you register the virtual machine to {SmartProxy}.
-Ensure that you configure port and firewall settings to open any necessary connections.
-
 ifndef::orcharhino[]
 For more information about port and firewall requirements, see {InstallingServerDocURL}satellite-ports-and-firewalls-requirements_{project-context}[Port and Firewall Requirements] in _{InstallingServerDocTitle}_ and {InstallingSmartProxyDocURL}capsule-ports-and-firewalls-requirements_{smart-proxy-context}[Ports and Firewalls Requirements] in _Installing {SmartProxyServer}_.
 endif::[]
+include::snip_prerequisite-configured-smart-proxy-registration-provisioning.adoc[]
 
 .Associating the `userdata` and `Cloud-init` Templates with the Operating System
 . In the {ProjectWebUI}, navigate to *Hosts* > *Operating Systems*, and select the operating system that you want to use for provisioning.
-. Click the *Template* tab.
-. From the *Cloud-init template* list, select *Cloudinit default*.
```

Comment:
There's no **Cloud-init template** list in the WebUI! Is it on purpose? 
There's only a single **Cloud-init template** (`CloudInit default`), so I'm guessing that there's nothing to configure and it's always used. 
@stejskalleos Can you please clarify this?

Reply:
There is a template called `CloudInit default`, after you associate it with the OS you can then select it as a `Cloud-init template` on the OS detail

![cloud](https://github.com/theforeman/foreman-documentation/assets/59385976/e3171480-eb79-430d-9cb9-d8bcdc796900)

Reply:
Ohhh, so there are steps missing! Thanks, this helped